src/commands/run.ts

```diff
@@ -32,7 +39,12 @@ export default class Run extends BaseExecutionCommand {
 
     this.validateInputs(Object.keys(flags).length, args.name)
 
-    if (flags['all-ms']) {
+    if (flags['all-mfe']) {
+      CliUx.ux.action.start(
+        `Running all micro frontends. Press ctrl + c to exit.`
+      )
+      await this.runAllComponents(Phase.Run, ComponentType.MICROFRONTEND)
+    } else if (flags['all-ms']) {
```

Comment:
If both `flags['all-mfe']` and `flags['all-ms']` are set, shouldn't it throw a specific error? If so, the `exclusive` flag property can be used for it.

Reply:
@cecchisandrone WDYT? using the exclusive flag property the error message would be something like this:
`--all-mfe= cannot also be provided when using --all-ms= `

Reply:
yes seems reasonable